src/ucs/stats/stats.h

```diff
@@ -105,24 +96,28 @@ void ucs_stats_node_free(ucs_stats_node_t *node);
 #define UCS_STATS_NODE_FREE(_node) \
     ucs_stats_node_free(_node)
 
+#define UCS_STATS_COUNTER(_node, _index) \
```

Comment:
Here is where we want to access ptr_array slots quickly outside ptr_array.[ch]

Reply:
can we make this an inline function ucs_ptr_array_elem() or ucs_ptr_array_at() etc?
 it can still have assert check for index (for debug mode)
(without leading __ since it's not internal macro)